### ISSUE TITLE
FocusInput Wiki

### ISSUE THREAD

**Issue by napcae** [philc/vimium#730]
Hi, 
first of all, thanks for this extension! It's very useful. As a new User to VIM and vimium, I had an issue with the shortcut "gi". The help only shows "Focus the first (or n-th) text box on the page". When you have more then one input box, the box you will write in will be highlighted. But I didn't know how to switch between the boxes. The solution is, use the tab key. After searching the changelog, I found it. 
**tl;dr:**
Make this function more present. Write for example "Focus the first (or n-th) text box on the page. Switch between boxes with tab."
